esp/scm/ws_workunits.ecm

```diff
@@ -151,6 +151,27 @@ ESPStruct [nil_remove] ECLQuery
     string ThorLog;
     [min_ver("1.35")] string QueryMainDefinition;
 };
+
+//  ===========================================================================
+
+ESPStruct [nil_remove] QuerySummaryStats
+{
+    string Endpoint;
+    string Status;
+    string StartTime;
+    string EndTime;
```

Comment:
Instead of echoing the start / end time - shouldn't this return the Time the state was taken?

IOW If the client is receiving an array of these stats, each row should have a Time stamp so we know at what time that stat was for?

Reply:
The Start/End time are returned from Roxie. @richardkchapman could you please answer this question?

Reply:
That sounds fine.